src/styles/components/_dropdown.scss

```diff
@@ -0,0 +1,50 @@
+@import '../abstracts/_abstracts-dir';
+
+.dropdown {
+  display: flex;
+  align-items: center;
+
+  &__arrow {
+    border-top: rem(4px) solid $dark;
+    border-right: rem(4px) solid transparent;
+    border-left: rem(4px) solid transparent;
+    margin-left: rem(8px);
+  }
+
+  &__menu {
+    position: absolute;
+    z-index: $zindex-dropdown;
+    right: 0;
+
+    box-shadow: rem(1px 1px 4px) $dropdown-list-shadow;
+    width: max-content;
+    padding: rem(2px) 0;
+    border: rem(1px) $shuttle-gray;
+    border-radius: rem(6px);
+    margin: rem(16px) 0 0;
+    background-color: $light;
+    cursor: pointer;
+  }
+
+  &__menu-item {
```

Comment:
The corner is pointing out 😅
![Uploading Screen Shot 2564-02-23 at 20.44.15.png…]()